LocalSettings.php

```diff
@@ -5449,6 +5486,7 @@
 	),
 	'wgGroupsRemoveFromSelf' => array(
 		'default' => array(),
+		'sau226wiki' => [ 'consul' => [ 'consul' ] ], //T2549
```

Comment:
Please use ) arrays

Reply:
I hope arrays like the ones below work. I have used a modified copy of an array below. Should hopefully work. Please re-review. Thank you very much